scripts/build/dev-build.ps1

```diff
@@ -60,6 +60,8 @@ try {
     Write-Host "Bin folder to use: $binPath"
     Write-Host "MSBuild: ${msbuildVersion}"
 
+    $StartDate=(Get-Date)
```

Comment:
It works 🎉 
The dev-build shows only 42 warnings now (CS0618, MSB3277).

To come back to the original idea of having clean build logs, we might consider putting the `/p:WarningLevel=0` argument back, as you originally did, to silence the useless noise.

Reply:
I would keep the build warnings, because ideally we should not have them. If we totally remove them from our logs, we'll forget they exist.

Most of the warnings are `'This rule is deprecated.` -related, and this is a good reminder that we should eventually just delete deprecated rules. Like we recently [removed S2758 ](https://github.com/SonarSource/sonar-dotnet/issues/3823), which had been deprecated for two years.

Reply:
I opened a [trello card](https://trello.com/c/RF2TaxeP/164-remove-rules-which-have-been-deprecated-before-sq-79-lts) to discuss it in the weekly.